financial_document/response_v1/invoice_to_string.rst

```diff
@@ -0,0 +1,34 @@
+Financial Document V1 Prediction
+================================
+:Filename:
+:Document type: INVOICE
+:Category: miscellaneous
+:Subcategory:
+:Locale: en; en; USD;
+:Invoice number: INT-001
+:Reference numbers: 2412/2019
+:Date: 2019-02-11
+:Due date: 2019-02-26
+:Time:
+:Supplier name: JOHN SMITH
+:Supplier address: 4490 Oak Drive Albany, NY 12210
+:Supplier company registrations:
+:Supplier payment details:
+:Customer name: JESSIE M HORNE
+:Customer address: 2019 Redbud Drive New York, NY 10011
+:Customer company registrations:
+:Tip:
+:Taxes:
+  +---------------+--------+----------+---------------+
+  | Base          | Code   | Rate (%) | Amount        |
+  +===============+========+==========+===============+
+  |               |        | 5.00     | 9.75          |
+  +---------------+--------+----------+---------------+
+:Total tax: 9.75
```

Comment:
I can see that it's also there in the tests, but the fact that it says `Total taxes` in almost all the others bothers me. Is this really necessary?